libbeat/docs/shared-command-line.asciidoc

```diff
@@ -39,6 +39,15 @@ Start http server for profiling. This option is useful for troubleshooting and p
 Write memory profile data to the specified output file. This option is useful for
 troubleshooting the Beat.
 
+*`-path.config`*::
```

Comment:
I haven't been following the discussion about these changes, which is probably good because I'm confused by some things that might also confuse users. I'm not sure what you mean here with some of these options. What are the data files that you're referring to? Is this a default location for the files you want Filebeat to read? And by setting the default location for configuration, do you mean mean that this option sets the default location for config files? And when you talk about miscellaneous files, can you be more specific? Maybe you are already planning to cover this in the other PR. :-)